hack/verify-container-images.sh

```diff
@@ -22,42 +22,20 @@ if [[ "${TRACE-0}" == "1" ]]; then
     set -o xtrace
 fi
 
-TRIVY_VERSION=0.34.0
-
-GO_OS="$(go env GOOS)"
-if [[ "${GO_OS}" == "linux" ]]; then
-  TRIVY_OS="Linux"
-elif [[ "${GO_OS}" == "darwin"* ]]; then
-  TRIVY_OS="macOS"
-fi
-
+VERSION=${1}
 GO_ARCH="$(go env GOARCH)"
```

Comment:
Interesting that we don't have this line in core CAPI and probably just assume that GOARCH is set?
Maybe something to fix in core CAPI? (@killianmuldoon)

Reply:
The `GO_ARCH` here is only used to give the resulting image a name - I'm not sure if it's used for anything else at this point - the ensure-trivy script sets its own `GO_ARCH`.

Maybe it's a good idea to just remove this here?

Reply:
I think it's not used to give it a name, it's used to match the name that is produced by make docker-build.

I think if we drop this line we either:
* have to pass it in
* have to find another way to match it correctly (I guess GOARCH might be different on macOS vs Linux (?))

I would avoid implicitly depending on a env variable being exported by someone calling that script

Reply:
You're right - I'll do a PR into CAPI to make them consistent and make CAPI's use of the GO_ARCH more explicit.